LottieSample/build.gradle

```diff
@@ -41,8 +41,8 @@ android {
     baseline file("lint-baseline.xml")
   }
   compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_7
-    targetCompatibility JavaVersion.VERSION_1_7
+    sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
Is this a binary compatible change? Will users of Lottie need to add anything to their config?

Reply:
@digitalbuddha this is just for the sample app. This would break binary compatibility of it were in the library though.
@SubhanAli94 Why did you change this though? This is a fully Kotlin project.

Reply:
Hi guys, Thanks for reviewing it. I was trying to build this project but suddenly some errors popped up saying to update this stuff and I did so. At the end, I thought it will be worth sharing it to all.

Reply:
@subhanAli94, what were the build errors?

Reply:
@gpeal I wrote exact error in question.